sdk/servicebus/service-bus/src/util/runtimeInfo.browser.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+interface NavigatorEx extends Navigator {
+  // oscpu is not yet standards-compliant, but can not be undefined in TypeScript 3.6.2
+  readonly oscpu: string;
+}
+
+/**
+ * Returns information about the platform this function is being run on.
+ * @ignore
+ * @internal
+ */
+export function getRuntimeInfo(): string {
+  const navigator = window.navigator as NavigatorEx;
+  const osInfo = {
+    key: "OS",
+    value: (navigator.oscpu || navigator.platform).replace(" ", "")
+  };
+
+  return `${[osInfo]}`;
```

Comment:
Won't this return `[object Object]`?

Reply:
Nice catch. I partly copied it from core-http and had only been testing the user agent string for the management client.
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/10092/commits/b3cc282cff3756d992e7b5a93b5bdfa592079a62